packages/block-library/src/navigation-menu/style.scss

```diff
@@ -59,19 +69,43 @@
 
 		// Sub-menus Flyout
 		& > li > ul {
-			margin: 0;
-			position: absolute;
-			background: #fff;
-			box-shadow: 0 0 8px 2px rgba(0, 0, 0, 0.2);
+			background: #000;
+			border-radius: 0.4rem;
 			left: 0;
-			top: 100%;
+			margin: 0;
 			min-width: max-content;
 			opacity: 0;
-			transition: all 0.5s ease;
+			position: absolute;
+			top: calc(100% + 1rem);
+			transition: opacity 0.15s linear, transform 0.15s linear, right 0s 0.15s;
+			transform: translateY(0.6rem);
 			visibility: hidden;
 
+			&::before {
```

Comment:
Most of these styles should likely be added within the `theme.scss` styles, which is what gets loaded when a theme does `add_theme_support( 'wp-block-styles' )`. Particularly the ones that are harder to replace, like `::before`.